<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win</class>
 <widget class="QDialog" name="Win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelWinPic">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>631</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="continuebutton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelprogress">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelLosePic">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MainMenubutton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MainMenu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Retrybutton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Retry Level</string>
   </property>
  </widget>
  <zorder>labelWinPic</zorder>
  <zorder>labelprogress</zorder>
  <zorder>labelLosePic</zorder>
  <zorder>continuebutton</zorder>
  <zorder>MainMenubutton</zorder>
  <zorder>Retrybutton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
